x-pack/solutions/observability/plugins/apm/public/components/app/onboarding/serverless_instructions.ts

```diff
@@ -19,36 +19,27 @@ import {
   createGoAgentInstructions,
   createDotNetAgentInstructions,
   createPhpAgentInstructions,
-  createOpenTelemetryAgentInstructions,
 } from './instructions';
 
 const DEFAULT_INSTRUCTION_TITLE = i18n.translate('xpack.apm.onboarding.defaultTitle', {
   defaultMessage: 'APM Agents',
 });
 
-function convertApmServerUrlToOtlpServiceUrl(apmServerUrl: string) {
-  if (!apmServerUrl) {
-    return '';
-  }
-
-  const urlParts = apmServerUrl.split('.');
-
-  return `${urlParts[0]}.ingest.${urlParts.slice(2).join('.')}:443`;
-}
-
 export function serverlessInstructions(
   {
     baseUrl,
     config,
     checkAgentStatus,
     agentStatus,
     agentStatusLoading,
+    managedOtlpServiceUrl,
   }: {
     baseUrl: string;
     config: ConfigSchema;
     checkAgentStatus: () => void;
     agentStatus?: boolean;
     agentStatusLoading: boolean;
+    managedOtlpServiceUrl: string;
```

Comment:
you pass down the `managedOtlpServiceUrl` here, but is it still consumed anywhere, since the otel instructions are gone?